docker-compose.yml

```diff
@@ -158,6 +158,20 @@ services:
       canton:
         condition: service_started
 
+  navigator:
+    image: digitalasset/daml-sdk:${SDK_VERSION}
+    container_name: daml_observability_navigator
+    ports:
+      - 4003:4000
+    command:
+      - "daml"
+      - "navigator"
+      - "server"
+      - "canton"
+      - "10011"
+    volumes:
+      - logs:/var/log/promtail
```

Comment:
Should we log something somewhere? If we don't care we can remove this volume.

I am not sure what is expected boostrap/configuration wise for the Navigator:
```
--feature-user-management <value>
By default, the login screen is now populated by quering the user mgmt service.
Disable to query party mgmt instead (the pre-2.0 default).
```

Reply:
Good point, will remove it